torch/distributed/_tensor/_shards_wrapper.py

```diff
@@ -0,0 +1,311 @@
+#!/usr/bin/env python3
```

Comment:
is this automatically added?

Reply:
removing, this is artifact from when it lived in torchrec